apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts

```diff
@@ -144,10 +145,11 @@ const transactionAddressValid = [
 ];
 
 const transactionE2E = [
-  /*{
-    transaction: new Transaction(Account.sep_ETH_1, Account.sep_ETH_2, "0.00001", Fee.SLOW), //todo: Reactivate when BE issue is fixed - LIVE-14844
+  {
+    transaction: new Transaction(Account.sep_ETH_1, Account.sep_ETH_2, "0.00001", Fee.SLOW),
     xrayTicket: "B2CQA-2574",
-  },*/
+    bugTicket: "LIVE-14844",
```

Comment:
you mentioned in the comment to the ticket that issue is fixed, so I guess we don't need bug ticket here anymore?